package.json

```diff
@@ -10,13 +10,18 @@
     "start": "nf start",
     "lint": "eslint --color *.js; exit 0;",
     "test:watch": "mocha -w",
-    "test": "istanbul cover -x *_[sS]pec.js _mocha  -- -R spec"
+    "test": "istanbul cover -x *_[sS]pec.js _mocha  -- -R spec",
+    "migrate": "sequelize db:migrate",
+    "migrate:undo:all": "sequelize db:migrate:undo:all",
```

Comment:
I don't think you need to remove the `migrate:undo` - I just don't know what it does.  This needs to be documented somewhere.  I still don't understand what the `migrate:test` command does either.